srv/components/provisioner/files/cluster_sls.template

```diff
@@ -20,12 +20,14 @@
 cluster:
   cluster_id: {{ grains["cluster_id"] }}
   storage_sets:
-    storage-set-1:
-      {% for num in range(num_of_nodes) %}
-      {% if 0 <= num <= 2 %}
+    {% for set_id in range(0, num_of_nodes, 3) %}
```

Comment:
I guess it's not actually  `set_id` but `first_node_id -1`. Could you elaborate variable names more...
as an option
```python
num_of_sets = num_of_nodes // 3 + bool(num_of_nodes % 3)
# not sure would it work in Jinja or not
```
so we can then build ranges over sets instead of nodes

Reply:
Renamed the variable